package.json

```
@@ -46,6 +46,7 @@
   },
   "devDependencies": {
     "@babel/core": "7.9.0",
+    "@fortawesome/fontawesome-free": "^5.13.0",
```

Pin to exact version 
```suggestion
    "@fortawesome/fontawesome-free": "5.13.0",
```